### PR TITLE
IGNITE-18710 Make default for delay to remove from logical topology non-zero

### PR DESCRIPTION
**Pull request by rpuch**
https://issues.apache.org/jira/browse/IGNITE-18710

### CODE REVIEW

File: modules/runner/src/integrationTest/java/org/apache/ignite/internal/BootstrapConfigTemplateMethod.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Defines name of a static method defined in a test class that will be invoked to produce a bootstrap config
+ * template to use intead of a default template.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD})
+public @interface BootstrapConfigTemplateMethod {
```
Comment (SammyVimes) [inline modules/runner/src/integrationTest/java/org/apache/ignite/internal/BootstrapConfigTemplateMethod.java:31]: This looks a bit like overkill imo

**Review by SammyVimes — APPROVED** [review]
OK overall, but I think the decision with annotation is questionable

File: modules/runner/src/integrationTest/java/org/apache/ignite/internal/BootstrapConfigTemplateMethod.java (hunk 1 above)
Comment (rpuch) [reply modules/runner/src/integrationTest/java/org/apache/ignite/internal/BootstrapConfigTemplateMethod.java]: I simplified the test and removed the annotation and the surrounding mechanism

**Comment by rpuch** [comment]
You are right, I simplified the code

**Review by SammyVimes — APPROVED** [review]
LGTM!

**Pull request merged**